Demo/Tweakable/TweakingOptionCell.swift

```diff
@@ -0,0 +1,57 @@
+import FinniversKit
+
+class TweakingOptionCell: UITableViewCell {
+    var option: TweakingOption? {
+        didSet {
+            titleLabel.text = option?.title
```

Comment:
We tend to create/use a method named something along the lines of `configure(withOption option: TweakingOption)` for configuring cells. Then this property can stay private